src/+otp/+kortewegdevries/invariant.m

```diff
@@ -0,0 +1,5 @@
+function I = invariant(u, Alpha, Nu, Rho, Dxf, dx)
```

Comment:
Don't you also need a Jacobian invariant vector product and Jacobian invariant adjoint vector product?

Reply:
We could, but it is unnecessary as it is numInvariants * numVars which will be reduced into a 1* numVars vector

Reply:
Don't you add the invariant jacobian adjoint vector product to the RHS?